Firestore/Source/Util/FSTLogger.m

```diff
@@ -19,22 +19,33 @@
 #import <FirebaseCore/FIRLogger.h>
 
 #import "Firestore/Source/API/FIRFirestore+Internal.h"
+#import "Firestore/Source/API/FIRFirestoreVersion.h"
 
 NS_ASSUME_NONNULL_BEGIN
 
+// assumes the first variable argument is the sdk version, and adds a space for it to the format
+// string.
+void logInternal(FIRLoggerLevel level, NSString* format, ...) {
```

Comment:
This should be static (to hide the symbol).

Also first letter should be capitalized http://google.github.io/styleguide/objcguide.html#function-names